### PR TITLE
Updating validation

### PR DESCRIPTION
**Pull request by jrichardBD**
@ekerwin @psantos1113 @stevebillings @akamen   Please review these code changes

### CODE REVIEW

**Comment by stevebillings** [comment]
Looks good.

**Comment by psantos1113** [comment]
looks good

File: src/main/java/com/blackducksoftware/integration/hub/builder/HubCredentialsBuilder.java [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+	public void setPassword(final String password) {
+		this.password = password;
+	}
+
+	public int getPasswordLength() {
+		return passwordLength;
+	}
+
+	/**
```
Comment (ekerwin) [inline src/main/java/com/blackducksoftware/integration/hub/builder/HubCredentialsBuilder.java]: probably fine for now, but we should consider a setEncryped and setDecrypted method and omit the setPassword and keep this passwordLength field completely internal to this class

File: src/main/java/com/blackducksoftware/integration/hub/builder/HubServerConfigBuilder.java [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+	private String password;
+	private int passwordLength;
+	private String proxyHost;
+	private String proxyPort;
+	private String proxyUsername;
+	private String proxyPassword;
+	private int proxyPasswordLength;
+	private String ignoredProxyHosts;
+
+	HubProxyInfo proxyInfo;
```
Comment (ekerwin) [inline src/main/java/com/blackducksoftware/integration/hub/builder/HubServerConfigBuilder.java]: these are package private for a legitimate reason?

**Comment by ekerwin** [comment]
Beautiful!

**Pull request merged**